src/components/OfflineWithFeedback.js

```diff
@@ -80,6 +80,14 @@ function applyStrikeThrough(children) {
     });
 }
 
+/**
+ * This method clears the errors for a specific chat
+ * @param {String} chatReportID
+ */
+function clearChatErrors(chatReportID) {
```

Comment:
I think we want to reuse these ones https://github.com/Expensify/App/blob/2356010ff7817ae27e603558b6905474929465d6/src/libs/actions/ReportActions.js#L4-L24 

Or if we cannot and need to add a new one we should add it to that file.

Reply:
I don't think this should go here, I think I jumped to soon, it should probably be where the OfflineWithFeedback is added, and then set as one of the close mehtods